pygromos/simulations/hvap_calculation/hvap_calculation.py

```diff
@@ -169,34 +169,34 @@ def run_liq(self):
         self.groSys_liq.rebase_files()
 
         #minsys_emin_liq, jobID
-        sys_emin_liq, jobID = simulation(in_gromos_system=self.groSys_liq, 
-                    project_dir=self.groSys_liq.work_folder,
-                    step_name="1_emin", 
-                    in_imd_path=self.imd_liq_min,
-                    submission_system=self.submissonSystem,
-                    analysis_script=simulation_analysis.do,
-                    verbose=self.verbose)
+        sys_emin_liq, jobID = simulation(in_gromos_system=self.groSys_liq,
+                                         project_dir=self.groSys_liq.work_folder,
+                                         step_name="1_emin",
+                                         in_imd_path=self.imd_liq_min,
+                                         submission_system=self.submissonSystem,
+                                         analysis_script=simulation_analysis.do,
+                                         verbose=self.verbose)
 
         #eq
-        sys_eq_liq, jobID = simulation(in_gromos_system=sys_emin_liq, 
-                    project_dir=self.groSys_liq.work_folder,
-                    step_name="2_eq", 
-                    in_imd_path=self.imd_liq_eq,
-                    submission_system=self.submissonSystem,
-                    analysis_script=simulation_analysis.do,
-                    verbose=self.verbose)
+        sys_eq_liq, jobID = simulation(in_gromos_system=sys_emin_liq,
+                                       project_dir=self.groSys_liq.work_folder,
+                                       step_name="2_eq",
+                                       in_imd_path=self.imd_liq_eq,
+                                       submission_system=self.submissonSystem,
+                                       analysis_script=simulation_analysis.do,
+                                       verbose=self.verbose)
 
         #md
-        sys_md_liq, jobID = simulation(in_gromos_system=sys_eq_liq, 
-                    project_dir=self.groSys_liq.work_folder,
-                    step_name="3_sd", 
-                    in_imd_path=self.imd_liq_md,
-                    submission_system=self.submissonSystem,
-                    analysis_script=simulation_analysis.do,
-                    verbose=self.verbose)
+        sys_md_liq, jobID = simulation(in_gromos_system=sys_eq_liq,
```

Comment:
Shouldn't jobID be removed here? @SchroederB

Reply:
this should be part of #117